service/pixelated/resources/root_resource.py

```diff
@@ -49,8 +51,10 @@
 
 
 class RootResource(BaseResource):
-    def __init__(self, services_factory):
+    def __init__(self, services_factory, static_folder=None):
         BaseResource.__init__(self, services_factory)
+        if static_folder:
+            set_static_folder(static_folder)
         self._public_static_folder = get_public_static_folder()
```

Comment:
@kalikaneko , before we reverted the changes you had proposed (again, sorry about that), we had something a bit simpler than global static_folder and a set_static_folder method. We simply had something like:

```
def __init__(self, services_factory, static_folder=get_public_static_folder()):
  self._public_static_folder = static_folder
```

Then, we don't need to make any of the changes to `service/pixelated/resources/__init__.py`. Does that make sense? We try to always avoid setters and global vars.

(pairing with @anikarni )